crowdsourcing/serializers/project.py

```diff
@@ -36,12 +35,20 @@ class ModuleSerializer(DynamicFieldsModelSerializer):
     deleted = serializers.BooleanField(read_only=True)
     template = TemplateSerializer(many=True, read_only=False)
     module_tasks = TaskSerializer(many=True, read_only=True)
+    file_id = serializers.IntegerField(write_only=True)
 
     def create(self, **kwargs):
         templates = self.validated_data.pop('template')
         project = self.validated_data.pop('project')
+        file_id = self.validated_data.pop('file_id')
```

Comment:
is the id attribute created by the file serializer unique? what if i upload the same file twice, would it produce different ids?